### ISSUE TITLE
Atom-beautify v0.29.0 fails to install through apm.  (now requires coffee-script)

### ISSUE THREAD

**Issue by kwv** [Glavin001/atom-beautify#891]
### Description

It appears that v0.29.0 depends on a coffee script build step.  I didn't previously have coffee-script installed globally, and upgrading from v028.

### Expected Results

```
$ apm install atom-beautify
Installing atom-beautify to C:\Users\user\.atom\packages failed

> spawn-sync@1.0.15 postinstall C:\Users\user\AppData\Local\Temp\apm-install-dir-11634-13808-hdq4p3\node_modules\atom-beautify\node_modules\cross-spawn\node_modules\spawn-sync
> node postinstall


> atom-beautify@0.29.0 postinstall C:\Users\user\AppData\Local\Temp\apm-install-dir-11634-13808-hdq4p3\node_modules\atom-beautify
> coffee script/build-options.coffee


npm WARN optional dep failed, continuing fsevents@1.0.11
npm WARN engine deep-extend@0.4.1: wanted: {"node":">=0.12.0","iojs":">=1.0.0"} (current: {"node":"0.10.40","npm":"2.13.3"})
npm WARN engine is-buffer@1.1.3: wanted: {"node":">=0.12"} (current: {"node":"0.10.40","npm":"2.13.3"})
npm WARN engine is-buffer@1.1.3: wanted: {"node":">=0.12"} (current: {"node":"0.10.40","npm":"2.13.3"})
npm WARN engine is-buffer@1.1.3: wanted: {"node":">=0.12"} (current: {"node":"0.10.40","npm":"2.13.3"})
npm WARN engine is-buffer@1.1.3: wanted: {"node":">=0.12"} (current: {"node":"0.10.40","npm":"2.13.3"})
npm WARN engine is-buffer@1.1.3: wanted: {"node":">=0.12"} (current: {"node":"0.10.40","npm":"2.13.3"})
npm WARN engine is-buffer@1.1.3: wanted: {"node":">=0.12"} (current: {"node":"0.10.40","npm":"2.13.3"})
npm WARN engine is-buffer@1.1.3: wanted: {"node":">=0.12"} (current: {"node":"0.10.40","npm":"2.13.3"})
'coffee' is not recognized as an internal or external command,
operable program or batch file.
npm ERR! Windows_NT 6.1.7601
npm ERR! argv "C:\\Users\\user\\AppData\\Local\\atom\\app-1.6.2\\resources\\app\\apm\\bin\\node.exe" "C:\\Users\\user\\AppData\\Local\\atom\\app-1.6.2\\resources\\app\\apm\\node_modules\\npm\\bin\\npm-cli.js" "--globalconfig" "C:\\Users\\user\\.atom\\.apm\\.apmrc" "--userconfig" "C:\\Users\\user\\.atom\\.apmrc" "install" "C:\\Users\\user\\AppData\\Local\\Temp\\d-11634-13808-lointr\\package.tgz" "--target=0.34.5" "--arch=ia32" "--msvs_version=2015"
npm ERR! node v0.10.40
npm ERR! npm  v2.13.3
npm ERR! code ELIFECYCLE

npm ERR! atom-beautify@0.29.0 postinstall: `coffee script/build-options.coffee`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the atom-beautify@0.29.0 postinstall script 'coffee script/build-options.coffee'.
npm ERR! This is most likely a problem with the atom-beautify package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     coffee script/build-options.coffee
npm ERR! You can get their info via:
npm ERR!     npm owner ls atom-beautify
npm ERR! There is likely additional logging output above.
```

### Steps to Reproduce

apm upgrade (failed)
apm uninstall atom-beautify
apm install atom-beautify

### Debug

```
npm i -g coffee-script
apm install atom-beautify
Installing atom-beautify to C:\Users\user\.atom\packages done
```

# Checklist

- [*] I have tried uninstalling and reinstalling Atom Beautify to ensure it installed properly
- [*] Searched for existing Atom Beautify Issues at https://github.com/Glavin001/atom-beautify/issues 
so I know this is not a duplicate issue
- [*] Generated debugging information and added link for `debug.md` Gist to this issue

**Comment by rochapablo** [comment]
It seems that I'm having the same problem.

```
> spawn-sync@1.0.15 postinstall C:\Users\Pablo\AppData\Local\Temp\apm-install-dir-11634-5268-y6rchd\node_modules\atom-beautify\node_modules\cross-spawn\node_modules\spawn-sync
> node postinstall


> atom-beautify@0.29.0 postinstall C:\Users\Pablo\AppData\Local\Temp\apm-install-dir-11634-5268-y6rchd\node_modules\atom-beautify
> coffee script/build-options.coffee


npm WARN engine deep-extend@0.4.1: wanted: {"node":">=0.12.0","iojs":">=1.0.0"} (current: {"node":"0.10.40","npm":"2.13.3"})
npm WARN optional dep failed, continuing fsevents@1.0.11
npm WARN engine is-buffer@1.1.3: wanted: {"node":">=0.12"} (current: {"node":"0.10.40","npm":"2.13.3"})
npm WARN engine is-buffer@1.1.3: wanted: {"node":">=0.12"} (current: {"node":"0.10.40","npm":"2.13.3"})
npm WARN engine is-buffer@1.1.3: wanted: {"node":">=0.12"} (current: {"node":"0.10.40","npm":"2.13.3"})
npm WARN engine is-buffer@1.1.3: wanted: {"node":">=0.12"} (current: {"node":"0.10.40","npm":"2.13.3"})
npm WARN engine is-buffer@1.1.3: wanted: {"node":">=0.12"} (current: {"node":"0.10.40","npm":"2.13.3"})
npm WARN engine is-buffer@1.1.3: wanted: {"node":">=0.12"} (current: {"node":"0.10.40","npm":"2.13.3"})
npm WARN engine is-buffer@1.1.3: wanted: {"node":">=0.12"} (current: {"node":"0.10.40","npm":"2.13.3"})
'coffee' is not recognized as an internal or external command,
operable program or batch file.
npm ERR! Windows_NT 6.2.9200
npm ERR! argv "C:\\Users\\Pablo\\AppData\\Local\\atom\\app-1.6.2\\resources\\app\\apm\\bin\\node.exe" "C:\\Users\\Pablo\\AppData\\Local\\atom\\app-1.6.2\\resources\\app\\apm\\node_modules\\npm\\bin\\npm-cli.js" "--globalconfig" "C:\\Users\\Pablo\\.atom\\.apm\\.apmrc" "--userconfig" "C:\\Users\\Pablo\\.atom\\.apmrc" "install" "C:\\Users\\Pablo\\AppData\\Local\\Temp\\d-11634-5268-1vuj756\\package.tgz" "--target=0.34.5" "--arch=ia32"
npm ERR! node v0.10.40
npm ERR! npm  v2.13.3
npm ERR! code ELIFECYCLE

npm ERR! atom-beautify@0.29.0 postinstall: `coffee script/build-options.coffee`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the atom-beautify@0.29.0 postinstall script 'coffee script/build-options.coffee'.
npm ERR! This is most likely a problem with the atom-beautify package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     coffee script/build-options.coffee
npm ERR! You can get their info via:
npm ERR!     npm owner ls atom-beautify
npm ERR! There is likely additional logging output above.
```

**Comment by Glavin001** [comment]
Ugh, Windows is always causing me problems... I'm on it. I'll try to release a fix later today. Thank you for reporting this issue.

**Comment by Glavin001** [comment]
Published patch to v0.29.1.

**Comment by kwv** [comment]
Thanks.  fixed it.
